lib/manageiq/messaging/stomp/client.rb

```diff
@@ -0,0 +1,50 @@
+module ManageIQ
+  module Messaging
+    module Stomp
+      class Client < ManageIQ::Messaging::Client
+        require 'stomp'
+        require 'manageiq/messaging/stomp/common'
+        require 'manageiq/messaging/stomp/queue'
+        require 'manageiq/messaging/stomp/background_job'
+        require 'manageiq/messaging/stomp/topic'
+
+        include Common
+        include Queue
+        include BackgroundJob
+        include Topic
+
+        private *delegate(:subscribe, :unsubscribe, :publish, :to => :stomp_client)
+        delegate :close, :to => :stomp_client
+        delegate :ack,   :to => :stomp_client
+
+        private
+
+        attr_reader :stomp_client
+
+        # options
+        #   :host
+        #   :username
+        #   :password
+        #   :port
+        #   :client_ref (optional)
+        #   :heartbeat  (optional, default to true)
+        def initialize(options)
+          host = { :host => options[:host], :port => options[:port] }
+          host[:passcode] = options[:password] if options[:password]
+          host[:login] = options[:username] if options[:username]
+
+          headers = {}
+          unless options[:heartbeat] == false
```

Comment:
avoid direct comparisons to `false`.  IMO, it's much clearer to explicitly show that nil is a "true" value as opposed to "not false"

````ruby
if options[:heartbeat].nil? || options[:heartbeat]
  headers.merge!...
```